docs/application/tizen-studio/release-notes/3-6-release-notes.md

```diff
@@ -0,0 +1,58 @@
+# Tizen Studio 3.6 Release Notes
+
+- Release Date: Nov 30, 2019
```

Comment:
Reminder comment. Please update to actual date.
With the date update, we will merge this PR and publish the pages.